components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -47,6 +51,8 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     return;
   }
 
+  [self removeObserversFromTabBarItems];
```

Comment:
You will also need to explicitly remove your observers from the deinit method until we drop iOS 10 support.

Reply:
deinit?

Reply:
do you mean `dealloc`?

Reply:
Added to dealloc. Let me know if there's another place it should be present.